### PR TITLE
add personal keymaps for jr bm60rgb

### PR DESCRIPTION
**Pull request by jrodan**
## Description

added individual keymap for bm60rgb.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bm60rgb/keymaps/jr/keymap.c [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "lib/lib8tion/lib8tion.h"
+
+#define TAPPING_TERM 350
+
```
Comment (zvecr) [inline keyboards/bm60rgb/keymaps/jr/keymap.c:20]: This need to go in a `config.h` for it to work
```suggestion

```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/bm60rgb/keymaps/jr/config.h [hunk 2]
```diff
@@ -0,0 +1,110 @@
+
```
Comment (drashna) [inline keyboards/bm60rgb/keymaps/jr/config.h:1]: Could you add a license header here, as well?

**Review by drashna — COMMENTED** [review]
If you wouldn't mind, could you change the folder name to match your github user name?  This makes it easier to identify and tag if there are issues in the future (eg, such as if there is a change that affects your keymap and we need to tag you)

**Comment by jrodan** [comment]
requested changes added - please have a look again

**Review by drashna — APPROVED** [review]

**Pull request closed without merging**